src/content/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions.mdx

```diff
@@ -772,7 +772,7 @@ Loss of signal settings include a time duration and a few actions.
   * Do not open "lost signal" incidents on expected termination. When a signal is expected to terminate, you can choose not to open a new incident. This is useful when you know that a signal will be lost at a certain time, and you don't want to open a new incident for that signal loss. The GraphQL node name for this is [`ignoreOnExpectedTermination`](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/#loss-of-signal).
 
 <Callout variant="important">
-  To prevent a loss of signal incident from opening when <DNT>**Do not open "lost signal" incident on expected termination**</DNT>, you must add the tag `termination: expected` to the entity. This tag tells us that we expected the signal to end. See [how to add the tag directly to the entity](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-tags).
+  To prevent a loss of signal incident from opening when <DNT>**Do not open "lost signal" incident on expected termination**</DNT>, you must add the tag `termination: expected` to the entity. This tag tells us that we expected the signal to end. See [how to add the tag directly to the entity](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-tags). Note that the tag `hostStatus: shutdown` will also prevent a prevent a loss of signal incident from opening. See [infrastructure host not reporting conditions.](/docs/infrastructure/infrastructure-alerts/create-infrastructure-host-not-reporting-condition/#create-condition)
```

Comment:
```suggestion
  To prevent a loss of signal incident from opening when <DNT>**Do not open "lost signal" incident on expected termination**</DNT>, you must add the tag `termination: expected` to the entity. This tag tells us that we expected the signal to end. See [how to add the tag directly to the entity](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-tags). Note that the tag `hostStatus: shutdown` will also prevent a prevent a "loss of signal" incident from opening. For more information, see [Create a "host not reporting" condition](/docs/infrastructure/infrastructure-alerts/create-infrastructure-host-not-reporting-condition/#create-condition).
```